src/hooks/useAiAssistant.ts

```diff
@@ -77,6 +77,7 @@ function useAiHook(integrationId: string, aiQuery: boolean, profileId?: string):
         return squid.ai().assistant(integrationId).profile(profileId).chat(question, aiAssistantOptions);
       }
     },
+    {},
```

Comment:
this should be default and no need to pass an empty object

Reply:
Done, however if a user is passing `deps` they may still need to pass an empty object, since both are optional.

I could also create a separate signature with `optionsOrDeps` as the second param, and check if its an `object` vs an `array`, but that might be overkill?